env/minimum-reqs.yml

```diff
@@ -0,0 +1,51 @@
+name: isce3-ci-sandbox
+channels:
+  - conda-forge
+  - nodefaults
+dependencies:
+  - backoff
+  # cmake: Current ver = 3.25
+  - cmake=3.18
+  - cxx-compiler
+  #####################
+  # EIGEN: Important
+  # Current ver: 3.4.0
```

Comment:
I think we can remove the comments like this, since they're not providing much useful information now that we've got a working minimum environment.

The `Current ver: 3.4.0` is not super useful because it's specific to a particular platform at a particular point in time. This version could change without warning as new packages versions are added to conda.

The `EIGEN: Important` part was useful initially to indicate which packages we thought might be most likely to introduce breakages in isce3 upon version changes (so we could try harder to pin those at their exact minimum supported version and be a little more flexible about the other packages). Now that we've got a minimum-reqs environment, though, I'm not sure that we need to persist this info in a comment.

There's not really a strict distinction between an "Important" dependency and an unimportant one. This was just my intuition about which dependencies we should prioritize.